src/backend/core/tests/test_models_teams.py

```diff
@@ -46,6 +46,12 @@ def test_models_teams_name_max_length():
         factories.TeamFactory(name="a " * 51)
 
 
+def test_models_teams_slug_empty():
+    """Slug field should not be empty."""
+    with pytest.raises(ValidationError, match="This field cannot be blank."):
+        models.Team.objects.create(slug="")
+
```

Comment:
you should test your save method override:
- the slug is automatically computed
- a slug that is already generated will not change if we change the name of the team and save it (this is what we want right?)

Reply:
I am not sure that what we want. Initially, we introduce the slug, to avoid having duplicated Team's name. Having a slug that doesn't reflect the current team's name break this check.

I agree that in general, slug doesn't change after their initial creation ... What should we do?

Reply:
To me, the slug is an unique version of the name. When the name changes, its slug should change accordingly. Ids are still unchangeable, if we want a secure way to access the team.

Reply:
Agree with changing the slug each time the name changes! Could be tested :wink: 
```
def test_models_teams_slug_updated_with_name():
    """The slug field should be updated when the name changes."""
    team = factories.TeamFactory()

    team.name = "new name"
    team.save()
    team.refresh_from_db()

    assert team.slug == "new-name"
```